app/views/candidate_interface/english_foreign_language/ielts/new.html.erb

```diff
@@ -3,12 +3,12 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class='govuk-heading-xl class='govuk-heading-xl''>
-      <span class="govuk-caption-xl"><%= t('page_titles.efl.start') %></span>
-      <%= t('page_titles.efl.ielts') %>
-    </h1>
-
     <%= form_with(model: @ielts_form, url: candidate_interface_ielts_path) do |f| %>
+      <%= f.govuk_error_summary %>
+      <h1 class='govuk-heading-xl class='govuk-heading-xl''>
```

Comment:
```suggestion
      <h1 class="govuk-heading-xl">
```

Reply:
For sure @frankieroberto, I think this is just legacy code so no problem changing!